src/main/resources/beam-template.conf

```diff
@@ -258,6 +258,11 @@ beam.replanning.Module_4 = "TimeMutator"
 beam.replanning.ModuleProbability_4 = 0.0
 beam.replanning.fractionOfIterationsToDisableInnovation = "double | Double.PositiveInfinity"
 
+#Skimmer
+beam.beamskimmer.writeObservedSkims = "boolean | true"
+beam.beamskimmer.writeAllModeSkimsForPeakNonPeakPeriods = "boolean | true"
+beam.beamskimmer.writeObservedSkimsPlus = "boolean | true"
```

Comment:
Is there a reason we are using booleans instead of an "interval" param like all of our other output controls?

Reply:
If no, then we should make this an interval where 0 is "false" and a positive integer means only write every nth interval. Also, at a minimum you should document what "observerdSkimsPlus" means right here in the config template so others can understand without having to read the source code.

Reply:
> Is there a reason we are using booleans instead of an "interval" param like all of our other output controls?

okay I will switch that to an integer

Reply:
> If no, then we should make this an interval where 0 is "false" and a positive integer means only write every nth interval. Also, at a minimum you should document what "observerdSkimsPlus" means right here in the config template so others can understand without having to read the source code.

I can document that, but it is soon going to be deprecated with the new skimmer coming along